package exception;

public class TestThrow1 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
	}
	public static double calcArea(int r){
		if(r<=0){
			throw new RuntimeException();
		}
		return r*r*Math.PI;
		
	}

}
